physics/docs/pdftxt/CLM_LAKE.txt

```diff
@@ -0,0 +1,59 @@
+/**
+\page CLM_LAKE_model CLM Lake Model 
+\section des_clmlake  Description
+
+The Community Land Model (CLM) lake model is a multi-level one-dimensional lake model that has been implemented within the operational 3-km HRRR and
+13-km RAP for small lakes (Benjamin et al. (2022) \cite gmd-15-6659-2022). This implementation is from the Community Land Model version 4.5 (Oleson et al. (2013) \cite Oleson2013). 
```

Comment:
Is this sentence correct about the version of the CLM? The previous wording was unclear, but I'm not 100% I captured the meaning correctly.

Reply:
The Oleson et al. 2013 is the technical description of the whole CLM version 4.5, the lake model being a part of it. Thus, this reference might be relevant.